firmware/kingst-la/sigrok-fwextract-kingst-la2016

```diff
@@ -192,5 +194,31 @@ if __name__ == "__main__":
     res = qt_resources(sys.argv[1])
 
     writer = res_writer(res)
-    writer.extract("fwfpga/LA2016A", "kingst-la2016a-fpga.bitstream")
-    writer.extract_re(r"fwusb/fw(.*)", r"kingst-la-\1.fw", decoder=maybe_intel_hex_as_blob)
+
+    '''
+    05-APR-2021
+    This extraction script has been tested with the
+    vendor software versions v3.5.0 and v3.5.1
+    These files were extracted and tested successfully:
+    saved 5430 bytes to kingst-la-01a2.fw (crc32=720551a9)
+    saved 178362 bytes to kingst-la2016a1-fpga.bitstream (crc32=7cc894fa)
+    saved 178542 bytes to kingst-la2016-fpga.bitstream (crc32=20694ff1)
+    saved 178379 bytes to kingst-la1016a1-fpga.bitstream (crc32=166866be)
+    saved 178151 bytes to kingst-la1016-fpga.bitstream (crc32=7db70001)
+    '''
+
+    # extract all firmware and fpga bitstreams
+    # writer.extract_re(r"fwfpga/(.*)", r"kingst-\1-fpga.bitstream")
+    # writer.extract_re(r"fwusb/fw(.*)", r"kingst-la-\1.fw", decoder=maybe_intel_hex_as_blob)
```

Comment:
If this is all commented out, can't it just go? We have the old code in the repository.

Reply:
If the manufacturer changes the firmware stream names in the resource file, these lines will be useful to discover those new names, and any other firmwares within the resource file. So I recommend to leave this commented code.

Reply:
I don't fully get it, but I have no strong opinion about it. Maybe a comment explaining why we keep commented out code is useful?